src/styles.scss

```diff
@@ -6,3 +6,13 @@
 .cursor-pointer {
   cursor: pointer;
 }
+
+.custom-tooltip {
+  .tooltip-inner {
+    background-color: white;
+    color: black;
+  }
```

Comment:
Can you replace this block with the following bootstrap classes? 
`class="bg-white text-dark"`

More info here: https://getbootstrap.com/docs/4.5/utilities/colors/

Reply:
@Utwo That can be done by setting the `encapsulation: ViewEncapsulation.None` for `app-icon-toggle-input` component. Let me know your inputs.

Reply:
@Utwo  We need to override the `.tooltip-inner` class of [**NgbTooltip**](https://ng-bootstrap.github.io/#/components/tooltip/api) So, we cannot directly apply the boostrap classes. Let me know your inputs.

Reply:
You are right! Thanks!